modules/ROOT/pages/procedures.adoc

```diff
@@ -1105,7 +1105,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cypher-manual/{page-ver
 | *Mode* 3+| WRITE
 |===
 
-[role=label--enterprise-edition label--admin-only]
+[role=label--enterprise-edition label--admin-only label--deprecated-2025.01]
```

Comment:
Not to forget: to add labels related to new Cypher versioning. Or add a note saying the following:
Cypher 5.26 (LTS) - the procedure is available and not deprecated
Cypher 5 in Neo4j 2025.01 - The procedure is deprecated
Cypher 25 in Neo4j 2025.01 - The procedure doesn’t exist